Makefile

```diff
@@ -1,9 +1,12 @@
 NB_CORES := $(shell grep --count '^processor' /proc/cpuinfo)
 PYLINT_DISABLE:= all
 PYLINT_ENABLE := F,E,unreachable,duplicate-key,unnecessary-semicolon,global-variable-not-assigned,unused-variable,binary-op-exception,bad-format-string,anomalous-backslash-in-string,bad-open-mode
```

Comment:
I would also add `trailing-whitespace`, `unneeded-not`, `singleton-comparison`. 

And `unused-import` can also be used here and not just on the tests.

Reply:
Ah. I thought we have a reason to ignore `unused-import`. Happy to see that we can add all of these.

Note that we fail on `global-variable-not-assigned` in gef.py. But also, we don't fail the tests when the linter fails. We probably should.